model-api/build.gradle.kts

```diff
@@ -58,6 +59,25 @@ kotlin {
     }
 }
 
+val generateVersionVariable by tasks.creating {
+    doLast {
+        val outputDir = project.layout.buildDirectory.dir("version_gen/org/modelix/model/api").get().asFile
+        outputDir.mkdirs()
+        outputDir.resolve("Version.kt").writeText(
+            """
+            package org.modelix.model.api
+
+            const val MODELIX_VERSION: String = "$version"
+
+            """.trimIndent(),
+        )
+    }
+}
```

Comment:
Putting this in the model-api project might be problematic, if we ever decide to start versioning model-api and model-server independently from each other. Should we move it to the model-server and call it sth like `MODELIX_SERVER_VERSION`?

Reply:
I will add the generator programmatically to each sub-project.
Importing the version might be problematic if gradle resolves versions differently.